### ISSUE TITLE
Move script in new location with namespace changing

### ISSUE THREAD

**Issue by BachiMjavanadze** [microsoft/vscode-dotnettools#74]
Please, in `Solution Expolrer` add the ability to drag and drop scripts to the new location (folder/project`*`) and change the namespace of the script accordingly. Also it should be possible to move the folder with scripts to a new location with namespace changing for all scripts.

`*` in `VisualStudioIDE` when a script is moved to new project, namespace is not changing. Hope `VScode` will do this job correctly.

**Comment by smitpatel** [comment]
Related #127 & #182

**Comment by webreidi** [comment]
Drag and drop is complete, closing this one as duplicate of https://github.com/microsoft/vscode-dotnettools/issues/127, for the remaining part of the ask for it to change the namespace upon move.

**Comment by BachiMjavanadze** [comment]
> Drag and drop is complete, closing this one as duplicate of [#127](https://github.com/microsoft/vscode-dotnettools/issues/127), for the remaining part of the ask for it to change the namespace upon move.

No, it doesn't work correctly; I created 2 projects. From the class library I moved the class by drag and drop to another project. The namespace didn't change:

![Image](https://github.com/user-attachments/assets/5d2c0de9-eeb6-4e7a-a0f5-a503579f3cb3)

**Comment by webreidi** [comment]
The namespace not changing when you move a class is being tracked under https://github.com/microsoft/vscode-dotnettools/issues/127.
